backend/plugin/setup.go

```diff
@@ -0,0 +1,55 @@
+package plugin
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/pprof"
+	"os"
+
+	hclog "github.com/hashicorp/go-hclog"
+)
+
+// SetupPluginEnvironment will read the environment variables and apply the
+// standard environment behavior.  As the SDK evolves, this will likely change!
+func SetupPluginEnvironment(pluginID string) hclog.Logger {
+	pluginLogger := hclog.New(&hclog.LoggerOptions{
```

Comment:
Why do we create a new logger and return it? Shouldn't this function receives a logger as param or use the global logger?

Reply:
I want to have all the standard setup that should be the same across every plugin in one place.  Currently each plugin does this on its own.  

The intention is this is called as the first line of "main()" and gives you the logger configured however we decide is best.  (Maybe the log level comes from environment?)